experimental/dds/tree/src/test/Forest.perf.tests.ts

```diff
@@ -17,8 +17,10 @@ import { refreshTestTree } from './utilities/TestUtilities';
 
 describe('Forest Perf', () => {
 	const testTree = refreshTestTree();
+	// Larger sizes can slow down correctness test runs, or even time out, so only run smaller sizes as correctness tests.
+	const sizes = isInPerformanceTestingMode ? [100, 1_000, 10_000, 100_000] : [100, 1_100];
```

Comment:
the last value in the non-perf path is 1100, was that intentional?

Reply:
no, thats a typo. Fixed.